examples/yum_collection_migration_spec.yml

```diff
@@ -0,0 +1,40 @@
+# a draft of a spec file to migrate content from core to a collection, for test purposes
+
+yum_collection:
+  action:
+    yum.py
+  module_utils:
+    yumdnf.py
+  modules:
+    packaging/os/yum.py
+  units:
+    test_yum.py
+  targets:
+    yum
+
+  # adding more to test the rest of plugin types
+  # missing networking ones for now
+  become:
+    sudo.py
+  cache:
+    memcached.py
+  callback:
+    default.py
+  connection:
+    local.py
+  docs_fragments:
+    shell_common.py
+  filter:
+    core.py
+  inventory:
+    ini.py
+  lookup:
+    env.py
+  test:
+    core.py
+  shell:
+    sh.py
+  strategy:
+    free.py
+    linear.py
```

Comment:
This will produce `free.py linear.py`. We saw somebody being misled by this YAML syntax just yesterday on the triage call...

Did you mean a YAML sequence or a mapping?

Reply:
Heh, good catch. I guess it doesn't matter which one we use? I'd say a list might be 'nicer', maybe?

Reply:
(It was meant to be a mapping but of course I forgot `:`)

Reply:
the advantage of 'mapping' is that it will automatically squash duplicates,  which is limited use since we also need to detect this globally. i'm fine either way,